feeds/wlan-ap/wlan-ap-config/files/etc/uci-defaults/99-tip-data

```diff
@@ -28,9 +91,32 @@ esac
 
 MODEL=$(echo $MODEL | sed "s/.*,//" | tr [a-z] [A-Z])
 
+# fallback check to get the serial number from mac address if flash does not contain this info.
+if [ ! $SERIAL ]; then
+	SERIAL=$(cat /sys/class/net/eth0/address | tr -d :)
+fi
+
+
+# Generating QR code from individual parameters.
+if [ ! QR_CODE ]; then
+	QR_CODE={"DT":"AP","DM":"$ID","VN":"$MANUF_ABBR","SN":"$SERIAL","MN":"$MODEL","HW":"$MODEL_REV"}
+fi
+
 uci set system.tip=tip
 uci set system.tip.serial="${SERIAL}"
 uci set system.tip.model="${MODEL}"
 uci set system.tip.platform="${PLATFORM}"
 uci set system.tip.firmware='0.1.0'
+uci set system.tip.sku_number="${SKU}"
+uci set system.tip.revision="${MODEL_REV}"
+uci set system.tip.model_description="${MODEL_DESCR}"
+uci set system.tip.manufacturer_name="${MANUF_NAME}"
+uci set system.tip.manufacturer_date="${MANUF_DATE}"
+uci set system.tip.manufacturer_url="${MANUF_URL}"
+uci set system.tip.reference_design="${REF_DESIGN}"
+uci set system.tip.certification_region="${CERT_REGION}"
+uci set system.tip.id="${ID}"
+uci set system.tip.manuf_abbr="${MANUF_ABBR}"
+uci set system.tip.qr_code="${QR_CODE}"
```

Comment:
Does this QR code get used?  There is no target function to retrieve it and it is constructed again in fill_entity_data().

Reply:
Thanks, Rick. I will remove this redundancy.
This got leftover when I moved the re-construction of the QR code from 99-tip-data to fill_entity_data().

Reply:
Thx